src/hex_map/hex_map.cpp

```diff
@@ -357,9 +357,9 @@ void HexMap::set_cell_item(const HexMapCellId &cell_id,
     if (p_item >= 0) {
         // set the cell
         Cell cell = {
-            .item = static_cast<unsigned int>(p_item),
-            .rot = static_cast<unsigned int>(p_rot),
-            .visible = true,
+            /* .item = */ static_cast<unsigned int>(p_item),
+            /* .rot = */ static_cast<unsigned int>(p_rot),
+            /* .visible = */ true,
```

Comment:
Alright, I need to think about this one.  Most likely I'll have to stick to c++17 for the time being, but I want to understand how long godot is going to stick to just that standard.  I love designated initializers for readability reasons.